config/locales/de.yml

```diff
@@ -1,79 +1,139 @@
+---
 de:
   activerecord:
-    errors:
-      models:
-        user:
-          attributes:
-            email:
-              blank: darf nicht leer sein
-              taken: ist schon vergeben
-              invalid: ist ungültig
-            role:
-              inclusion: ist nicht in der Liste enthalten
-            password:
-              blank: darf nicht leer sein
-              too_short: zu kurz (min. 6 Zeichen)
-            password_confirmation:
-              confirmation: Die eingegebenen Passwörter stimmen nicht überein
-        profile:
-          attributes:
-            first_name:
-              blank: darf nicht leer sein
-            last_name:
-              blank: darf nicht leer sein
-            avatar_content_type:
-              invalid: ist ungültig
-    models:
-      client: Klient
-      profile: Profil
-      user: Benutzer
     attributes:
       client:
-        first_name: Vorname
-        last_name: Nachname
+        city: Stadt
+        comments: Bemerkungen
+        competent_authority: Involvierte Stelle
         date_of_birth: Geburtsdatum
-        nationality: Nationalität
-        permit: Bewilligung
+        deleted_at: Gelöscht
+        education: Ausbildung
+        email: Email
+        first_name: Vorname
         gender: Geschlecht
-        street: Strasse
-        zip: Postleitzahl
-        city: Stadt
-        phone: Telefonnummer
         goals: Ziele
-        education: Ausbildung
         hobbies: Hobbies
         interests: Interessen
-        state: Status
-        comments: Bemerkungen
-        competent_authority: Involvierte Stelle
         involved_authority: Fallführende Stelle
+        language_skills: Sprachkenntnisse
+        last_name: Nachname
+        nationality: Nationalität
+        permit: Bewilligung
+        phone: Telefonnummer
+        relatives: Verwandte
+        schedules: Fahrpläne
+        state: Status
+        street: Strasse
+        user: :activerecord.models.user
+        zip: Postleitzahl
+      contact:
+        city: Ort
+        contact_emails: Mailadressen
+        contact_phones: Telefonnummern
+        extended: Adresszusatz
+        name: Name
+        postal_code: PLZ
+        street: Strasse
+      contact_email:
+        body: Email
+        contact: :activerecord.models.contact
+        label: Art
+      contact_phone:
+        body: Telefonnummer
+        contact: :activerecord.models.contact
+        label: Art
+      department:
+        associated_users: Assozierte Benutzer
+        city: Ort
+        contact: :activerecord.models.contact
+        department_manager: Standort Betreuer
+        name: Name
+        user: :activerecord.models.user
       language_skill:
+        client: :activerecord.models.client
```

Comment:
Possibly won't be needed, as language_skill will also be connected with the volunteer.